jazz-ui/src/app/pages/service-access-control/service-access-control.component.ts

```diff
@@ -106,6 +106,9 @@ export class ServiceAccessControlComponent implements OnInit {
     } 
   }
   
+  refresh(){
```

Comment:
is this required if its not doing anything?

Reply:
every component has a refresh function, We might be needing this in future when the component is implemented

Reply:
so should this be added in future then rather than now.